why/vs/readme.md

```diff
@@ -76,7 +76,7 @@ If you find something invalid or out of date in the comparisons, please [create
             <td>❌ 🤓</td>
             <td>❌</td>
             <td>❌</td>
-            <td>❌</td>
+            <td>✅</td>
```

Comment:
mmm sort of. You can get things to get notified about things over sockets but you can't send data over sockets. I feel like it's more of half check.

Thanks for the PR though.